datasets/monero/monero.py

```diff
@@ -0,0 +1,198 @@
+# coding=utf-8
+# Copyright 2020 The HuggingFace Datasets Authors and the current dataset script contributor.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""MoNERo: a Biomedical Gold Standard Corpus for the RomanianLanguage"""
+
+from __future__ import absolute_import, division, print_function
+
+import logging
+import os
+
+import datasets
+
+
+# Find for instance the citation on arxiv or on the dataset repo/website
+_CITATION = """\
+@inproceedings{inproceedings,
+author = {Mitrofan, Maria and Mititelu, Verginica and Grigorina, Mitrofan},
+year = {2019},
+month = {01},
+pages = {71-79},
+title = {MoNERo: a Biomedical Gold Standard Corpus for the Romanian Language},
+doi = {10.18653/v1/W19-5008}
+}
+"""
+
+# You can copy an official description
+_DESCRIPTION = """\
+ MoNERo comprises 154,825morphologically annotated tokens and 23,188entity annotations belonging to four entity se-mantic  groups  corresponding  to  UMLS  Se-mantic Groups.
+"""
+
+_HOMEPAGE = "https://slp.racai.ro/index.php/resources/monero-3/"
+
+_LICENSE = "MIT License"
+
+# The HuggingFace dataset library don't host the datasets but only point to the original files
+# This can be an arbitrary nested dict/list of URLs (see below in `_split_generators` method)
+_URL = "https://raw.githubusercontent.com/iliemihai/MoNERo/main/"
+_CARDIOLOGY_FILE = "MoNERo_cardiology.txt"
+_DIABETES_FILE = "MoNERo_diabetes.txt"
+_ENDOCRINOLOGY_FILE = "MoNERo_endocrinology.txt"
+_MONERO_FILE = "MoNERo.txt"
+
+
+class MONEROConfig(datasets.BuilderConfig):
+    """BuilderConfig for RONEC dataset"""
+
+    def __init__(self, **kwargs):
+        super(MONEROConfig, self).__init__(**kwargs)
+
+
+class MONERODataset(datasets.GeneratorBasedBuilder):
```

Comment:
```suggestion
class Monero(datasets.GeneratorBasedBuilder):
```
this must be the camel caser version of the dataset script name `monero`